app/components/modal/FirstModal/FirstModal.module.css

```
@@ -0,0 +1,107 @@
+.firstmodalOverlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 1000;
+  font-weight: bold;
+}
+
+.firstmodalContent {
+  background-color: #ffffff;
+  border-radius: 15px;
+  padding: 40px;
+  width: 500px;
+  max-width: 90%;
+  text-align: center;
+  position: relative;
+}
+
+.closeButton {
+  position: absolute;
+  top: 0px;
+  right: 22px;
+  background: none;
+  border: none;
+  font-size: 36px;
+  cursor: pointer;
+  color: #7b90a6;
+}
+
+.title {
+  font-size: 24px;
+  margin-bottom: 10px;
+  font-weight: bold;
+  color: #333;
+}
+
+.description {
+  text-align: left;
+  font-size: 14px;
+  margin-bottom: 10px;
+  color: #2c3844;
+}
+
+.apititle {
```

[キャメルケース](https://developer.mozilla.org/ja/docs/Glossary/Camel_case)に統一でお願いします！
```suggestion
.apiTitle {
```